api/memo_history.go

```diff
@@ -0,0 +1,26 @@
+package api
+
+type MemoHistory struct {
+	ID int `json:"id"`
+
+	MemoID    int   `json:"memoId"`
+	CreatedTs int64 `json:"createdTs"`
+
+	// Domain specific fields
+	Content string `json:"content"`
+}
```

Comment:
The edit history I proposed is to prevent accidental deletion of the texts, I think its unusual for someone to accidentally delete a resource in the memo, and if they do so it's still available from `/api/resource`. I suggest to keep it simple